packages/account/src/providers/provider.ts

```diff
@@ -460,8 +483,17 @@ export default class Provider {
    * @returns A promise that resolves to a Provider instance.
    */
   static async create(url: string, options: ProviderOptions = {}): Promise<Provider> {
-    const provider = new Provider(url, options);
+    const [urlToUse, addBasicAuth] = this.handleBasicAuth(url);
```

Comment:
According to the previous suggestions:

```suggestion
    const { url: urlToUse, auth } = this.extractBasicAuth(url);
```